src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -107,50 +118,11 @@ class ObjectField extends React.Component {
     };
   }
 
-  onPropertyBlur(name) {
-    return (path = []) => {
-      this.props.onBlur([name].concat(path));
-    };
-  }
-
-  getStateFromProps(props) {
-    const { schema, formData, registry } = props;
-    return getDefaultFormState(schema, formData, registry.definitions) || {};
-  }
-
-  // This runs a series of steps that order properties and then group them into
-  // expandable groups. If there are no expandable groups, then the end result of this
-  // will be an array of single item arrays
-  orderAndFilterProperties(schema, uiSchema) {
-    const properties = Object.keys(schema.properties);
-    const orderedProperties = orderProperties(
-      properties,
-      _.get('ui:order', uiSchema),
-    );
-    const filteredProperties = orderedProperties.filter(
-      prop => !schema.properties[prop]['ui:hidden'],
-    );
-    const groupedProperties = _.groupBy(item => {
-      const expandUnderField = _.get(
-        [item, 'ui:options', 'expandUnder'],
-        uiSchema,
-      );
-      return expandUnderField || item;
-    }, filteredProperties);
-
-    return _.values(groupedProperties);
-  }
-
   isRequired(name) {
     const { schema } = this.props;
-    const schemaRequired =
-      Array.isArray(schema.required) && schema.required.indexOf(name) !== -1;
-
-    if (schemaRequired) {
-      return schemaRequired;
-    }
-
-    return false;
+    return (
+      Array.isArray(schema.required) && schema.required.indexOf(name) !== -1
+    );
```

Comment:
This is just simplified from the following, where `isTrue` represents the returned value on line 124:

```js

const isTrue = true;

if (isTrue) {
  return isTrue;
}

return false
```